subsys/bluetooth/shell/csis.c

```diff
@@ -258,3 +203,36 @@ SHELL_STATIC_SUBCMD_SET_CREATE(csis_cmds,
 
 SHELL_CMD_ARG_REGISTER(csis, &csis_cmds, "Bluetooth CSIS shell commands",
 		       cmd_csis, 1, 1);
+
+ssize_t csis_ad_data_add(struct bt_data *data, const size_t data_size, const bool discoverable)
```

Comment:
Looks like this was actually added, but never used :D

Reply:
It is, in `ad_init`

Reply:
Sorry, must've failed my search last time :D